src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+ * Add a 3rd argument to `ParameterBag::getInt`. When set to `true`, the method throw an exception if the parameter is invalid
+ * Deprecate calling `ParameterBag::getInt` without setting `true` as 3rd argument
+ * Add `ParameterBag::getString` to convert a parameter into string and throw an exception if the parameter is invalid
+ * Calling `ParameterBag` `getDigit`, `getAlnum`, `getAlpha` on an `array` throws an `UnexpectedValueException` instead of a `TypeError`
```

Comment:
please add `()` as a suffix to all method named.

also, did you mean `ParameterBag::getDigit()` ? (missing `::`)

Reply:
I want to list the 3 methods on the same class but I don't want to repeat the class name.

Reply:
Calling `ParameterBag::getDigit()`, `getAlnum()` and `getAlpha()` on an `array` throws an `UnexpectedValueException` instead